### PR TITLE
WeatherFlow Cloud - Adding Sensor Documentation

### PR DESCRIPTION
**Pull request by jeeftor**
## Proposed change

As we add sensors to the integration we need docs!

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/111651
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Clarified functionality and usage between WeatherFlow and WeatherFlow Cloud integrations.
  - Updated categorization and added detailed sensor descriptions for the WeatherFlow Cloud integration.

### CODE REVIEW

File: source/_integrations/weatherflow_cloud.markdown [hunk 1]
```diff
@@ -18,3 +20,16 @@ ha_integration_type: hub
 The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
 
 {% include integrations/config_flow.md %}
+
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow_cloud.markdown:30]:
```suggestion
```
Remove the extra blank line

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/weatherflow_cloud.markdown [hunk 2]
```diff
@@ -18,3 +20,15 @@ ha_integration_type: hub
...
+## Sensors
+
+Additionally this integration provides access to the following sensors
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
Additionally, this integration provides access to the following sensors
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 3]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
+### Which integration should I use
+
+The [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) integration provides local communications with your weather station via UDP. It returns raw sensor data readings.
+The [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) integration provides a data feed reflective of what is presented in the Tempest mobile application - including Forecast data. WeatherFlow explains the differences in their FAQ:
+
+> Can I access data from the hardware locally? To ensure access to the best data, all third-party applications and integrations should use the remote interfaces (REST & Websockets) as their primary source for data, even if they are running on the same network as the local Tempest device. There is also a local UDP interface available for those applications that require completely off-grid applications, but this should only be used as a backup to the remote interfaces.
+
 {% include integrations/config_flow.md %}
 
+### Temperature Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
### Temperature sensors
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 4]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
+| --- | --- |
+| Dew point | The temperature to which air must be cooled to become saturated with water vapor. |
+| Feels like | An index that combines temperature and humidity to determine the apparent temperature. |
+| Heat index | A measure of how hot it feels when relative humidity is factored in with the actual air temperature. |
+| Temperature | The degree or intensity of heat present in the area. |
+| Wet bulb globe temperature | A composite temperature used to estimate the effect of temperature, humidity, wind speed, and solar radiation on humans. |
+| Wet bulb temperature | The lowest temperature that can be reached under current ambient conditions by the evaporation of water only. |
+| Wind chill | The lowering of body temperature due to the passing-flow of lower-temperature air. |
+
+### Air Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
### Air sensors
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 5]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
+
+| Sensor | Description |
+| --- | --- |
+| Air density | The mass per unit volume of Earth's atmosphere. |
+| Pressure barometric | The pressure exerted by the atmosphere at the earth's surface. |
+| Pressure sea level | The atmospheric pressure at mean sea level. |
+| Pressure station | The atmospheric pressure at the station level. |
+
+### Lightning Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
### Lightning sensors
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @jeeftor :+1:

**Comment by c0ffeeca7** [comment]
@klaasnicolaas, are you good with the changes?

File: source/_integrations/weatherflow.markdown [hunk 6]
```diff
@@ -22,6 +22,13 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
...
 {% endnote %}
 
+### Which integration should I use
+
+The [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) integration provides local communications with your weather station via UDP. It returns raw sensor data readings.
+The [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) integration provides a data feed reflective of what is presented in the Tempest mobile application - including Forecast data. WeatherFlow explains the differences in their FAQ:
+
+> Can I access data from the hardware locally? To ensure access to the best data, all third-party applications and integrations should use the remote interfaces (REST & Websockets) as their primary source for data, even if they are running on the same network as the local Tempest device. There is also a local UDP interface available for those applications that require completely off-grid applications, but this should only be used as a backup to the remote interfaces.
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow.markdown]: If this is intended as a FAQ rather than a quote then I would display this differently.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/weatherflow.markdown (hunk 6 above)
Comment (jeeftor) [reply source/_integrations/weatherflow.markdown]: How do you suggest I change it up?

Comment (klaasnicolaas) [reply source/_integrations/weatherflow.markdown]: Just a header (question) with the paragraph (answer) underneath. Furthermore, I would describe the answer as straightforwardly as possible.

./Klaas

File: source/_integrations/weatherflow_cloud.markdown [hunk 7]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
 
+### Which integration should I use
+
+The [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) integration provides local communications with your weather station via UDP. It returns raw sensor data readings.
+The [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) integration provides a data feed reflective of what is presented in the Tempest mobile application - including Forecast data. WeatherFlow explains the differences in their FAQ:
+
+> Can I access data from the hardware locally? To ensure access to the best data, all third-party applications and integrations should use the remote interfaces (REST & Websockets) as their primary source for data, even if they are running on the same network as the local Tempest device. There is also a local UDP interface available for those applications that require completely off-grid applications, but this should only be used as a backup to the remote interfaces.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]: It took me a moment to understand the answer here. I would rephrase this to something like....: Yes, there is a local UDP interface...but.... To ensureaccess....
but that's my personal preference.

File: source/_integrations/weatherflow.markdown (hunk 6 above)
Comment (jeeftor) [reply source/_integrations/weatherflow.markdown]: This seem better? (tried to simplify it)

image

Comment (jeeftor) [reply source/_integrations/weatherflow.markdown]: I think I'm going with this one :)

File: source/_integrations/weatherflow_cloud.markdown (hunk 7 above)
Comment (jeeftor) [reply source/_integrations/weatherflow_cloud.markdown]: Check again - made some changes.

File: source/_integrations/weatherflow.markdown [hunk 8]
```diff
@@ -22,6 +22,14 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
 You may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app considers both forecasts and neighboring weather stations in addition to the local data used in this {% term integration %}.
 {% endnote %}
 
+### Which integration(s) should I use
+
+There are two integrations for WeatherFlow devices, and you are not limtied to selecting just one.
+
+- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP` based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow.markdown]:
```suggestion
- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```

File: source/_integrations/weatherflow.markdown [hunk 9]
```diff
@@ -22,6 +22,14 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
...
+
+-  [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) is a *cloud* based integration that closely mirrors the data availabe via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow.markdown]:
```suggestion
-  [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) is a *cloud*-based integration that closely mirrors the data available via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data. 
```

File: source/_integrations/weatherflow.markdown [hunk 10]
```diff
@@ -22,6 +22,14 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
 You may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app considers both forecasts and neighboring weather stations in addition to the local data used in this {% term integration %}.
 {% endnote %}
 
+### Which integration(s) should I use
+
+There are two integrations for WeatherFlow devices, and you are not limtied to selecting just one.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow.markdown]:
```suggestion
There are two integrations for WeatherFlow devices, and you are not limited to selecting just one.
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 11]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
 
 The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station, you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
 
+### Which integration(s) should I use
+
+There are two integrations for WeatherFlow devices, and you are not limtied to selecting just one.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
There are two integrations for WeatherFlow devices, and you are not limited to selecting just one.
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 12]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
+
+- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP` based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 13]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
+
+-  [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) is a *cloud* based integration that closely mirrors the data availabe via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data.
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
-  [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) is a *cloud*-based integration that closely mirrors the data available via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data. 
```

File: source/_integrations/weatherflow.markdown [hunk 14]
```diff
@@ -22,6 +22,14 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
 You may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app considers both forecasts and neighboring weather stations in addition to the local data used in this {% term integration %}.
 {% endnote %}
 
+### Which integration(s) should I use
+
+There are two integrations for WeatherFlow devices, and you are not limited to selecting just one.
+
+- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
+
+-  [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) is a *cloud*-based integration that closely mirrors the data available via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data.
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow.markdown]:
```suggestion
- [WeatherFlow Cloud](/integrations/weatherflow_cloud) is a *cloud*-based integration that closely mirrors the data available via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data. 
```

File: source/_integrations/weatherflow.markdown [hunk 15]
```diff
@@ -22,6 +22,14 @@ The **WeatherFlow** {% term integration %} is a local-only {% term integration %
 You may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app considers both forecasts and neighboring weather stations in addition to the local data used in this {% term integration %}.
 {% endnote %}
 
+### Which integration(s) should I use
+
+There are two integrations for WeatherFlow devices, and you are not limited to selecting just one.
+
+- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow.markdown]:
```suggestion
- [WeatherFlow](/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 16]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
 
 The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station, you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
 
+### Which integration(s) should I use
+
+There are two integrations for WeatherFlow devices, and you are not limited to selecting just one.
+
+- [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
- [WeatherFlow](/integrations/weatherflow) is a *local only* `UDP`-based integration that will read data directly from the device. This integration does require the Home Assistant server and the WeatherFlow device to be on the same subnet.
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 17]
```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
...
+
+-  [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) is a *cloud*-based integration that closely mirrors the data available via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data.
```
Comment (klaasnicolaas) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
- [WeatherFlow Cloud](/integrations/weatherflow_cloud) is a *cloud*-based integration that closely mirrors the data available via the Weatherflow Tempest mobile applications and is likely a good starting place for most users as it provides both **Forecast** and **Sensor** data. 
```

**Review by klaasnicolaas — DISMISSED** [review]

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**